pkg/util/k8s/resource/configmap.go

```diff
@@ -36,8 +36,10 @@ func NewConfigMap(c client.Client, name, namespace string, labels map[string]str
 // Get returns an existing ConfigMap and an error if any.
 func (cm ConfigMap) Get() (*corev1.ConfigMap, error) {
 	configmap := &corev1.ConfigMap{}
-	err := cm.client.Get(context.TODO(), cm.NamespacedName, configmap)
-	return configmap, err
+	if err := cm.client.Get(context.TODO(), cm.NamespacedName, configmap); err != nil {
```

Comment:
I know it is out of the scope of the change, but are we able to replace TODO or we really don't know what to do with context here?

Reply:
Yes, it would be better to pass in a proper context, but this is a wider change to do consistently.  The k8s client function signature changed and this is in keeping with the existing code.  I'd prefer to leave as-is for now and we can do a follow-on PR if needed.